recipe/build.sh

```diff
@@ -1,13 +1,20 @@
 mkdir -p build
 cd build
 
-declare -a QT_SELECTOR
+QT_SELECTOR=()
 
 if [[ "$build_variant" = "qt5" ]]; then
-      QT_SELECTOR+=(-D SOQT_USE_QT5=ON)
+      QT_SELECTOR+=("-D SOQT_USE_QT5=ON")
 fi
 if [[ "$build_variant" = "qt6" ]]; then
-      QT_SELECTOR+=(-D SOQT_USE_QT6=ON)
+      QT_SELECTOR+=("-D SOQT_USE_QT6=ON")
```

Comment:
honestly, why bother with bash arrays...

```
QT_SELECTION=-DSOQT_USE_QT6=ON
```

then
```
cmake \
   ${QT_SELECTOR} \
```

Reply:
thanks, yes this is better